source/catalog.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>Каталог</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="header__nav nav">
+      <a class="nav__logo" href="#">
+        <img src="img/images/logo-1.svg" width="70" height="59" alt="логотип Cat Energy">
+        <img src="img/images/logo-2.svg" width="101" height="18" alt="cat energy">
+      </a>
+      <button class="header__button-header" type="button"><span class="visually-hidden">Открыть меню</span></button>
+    </nav>
+  </header>
+  <main class="main-inner">
+    <section class="main-inner__catalog-products catalog-products">
+      <h1 class="catalog-products__title">Каталог продукции</h1>
+      <div class="catalog-products__cards">
+        <ul class="catalog-products__card-list card-list">
+          <li class="card-list__card-item card-item">
+            <a class="card-item__card-link" href="">
+              <img src="img/images/img-catalog-1.jpg" width="140" height="107" alt="корм с курицей 500 г">
+              <h2 class="card-item__card-title">cat energy pro 500 г</h2>
+              <table class="card-description">
+                <dt>Масса</dt>
+                <dd>500 г</dd>
+                <dt>Вкус</dt>
+                <dd>Курица</dd>
+                <dt>Цена</dt>
+                <dd>700 ₽</dd>
+              </table>
+              <button class="card-item__button-catalog-card" type="button"><span>Заказать</span></button>
+            </a>
```

Comment:
У всех карточек так.

```suggestion
            <a class="card-item__card-link" href="">
              <img src="img/images/img-catalog-1.jpg" width="140" height="107" alt="корм с курицей 500 г">
            </a>
            <h2 class="card-item__card-title">
              <a href="#">cat energy pro 500 г</a>
            </h2>
            <table class="card-description">
            <tbody>
              <tr>
                <td>Масса</td>
                <td>500 г</td>
              </tr>
              <tr>
                <td>Вкус</td>
                <td>Курица</td>
              </tr>
              <tr>
                <td>Цена</td>
                <td>700 ₽</td>
              </tr>
            </tbody>
            </table>
            <a class="card-item__button-catalog-card" type="button">Заказать</a>
```